src/airport.js

```diff
@@ -16,11 +16,13 @@ const airport = {
         return this.landedPlanes.length < this.defaultAirportCapacity
 
     },
-
     planesExist: function (planes)
     {
-        return this.landedPlanes.includes(planes)
-
+        if (this.landedPlanes.includes(planes))
```

Comment:
I could have used an if condition in the removePlanes && landPlanes function to return the error messages, I think the code would have been cleaner and simpler.